operator_config.py

```diff
@@ -42,7 +42,14 @@
         "type": "integer",
         "coerce": int,
         "default": 1,
-        "help_message": "CPU cores for each available instance. Note: container instances only get CPU during request processing and startup. Read more https://cloud.google.com/run/docs/configuring/cpu-allocation",
+        "help_message": "CPU cores for each available instance. Note: container instances only get CPU during request processing and startup by default. Read more https://cloud.google.com/run/docs/configuring/cpu-allocation",
+    },
+    "cpu_always_allocated": {
+        "required": True,
+        "type": "boolean",
```

Comment:
should it be a required value?

Reply:
It doesn't make any difference I don't think 🤔 The only place `OPERATOR_SCHEMA` is actually used is in the `bentoctl init` command as far as I can see. I just followed the pattern of making everything required (even things which technically aren't).

Technically, anything which has a default defined in the terraform template isn't required even now!